src/components/SearchResults/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Results, Title, SubTitle } from "./styles";
+import SearchCard from "../SearchCard";
+
+export default function SearchResults({ media, isShowing }) {
+    const { movies, tvShows } = media;
+
+    const noResults = movies.length === 0 && tvShows.length === 0;
+
+    const movieList = (
+        <ul>
+            <Title>Movies</Title>
+            {movies.map((item) => (
+                <SearchCard key={item.id} details={item} type="movie" />
+            ))}
+        </ul>
+    );
+
+    const tvShowList = (
+        <ul>
+            <Title>TV Shows</Title>
+            {tvShows.map((item) => (
+                <SearchCard key={item.id} details={item} type="tv" />
+            ))}
+        </ul>
+    );
+
+    return (
+        <Results isShowing={isShowing} isEmpty={noResults}>
+            {movies.length !== 0 ? movieList : null}
```

Comment:
How bout:
```javascript
{!!movies.length && movieList}
```

movies.length can either results to 0 (false) or > 0 (true). When length is true, it will display the next value after the `&&` which is `movieList`. However, when it results to false, it will display `0`. To avoid this, we use double negation to convert it into a boolean. So `!!0` will become `false` in which it will not get rendered onto the screen.